src/panels/config/automation/action/types/ha-automation-action-sequence.ts

```diff
@@ -0,0 +1,61 @@
+import { CSSResultGroup, html, LitElement } from "lit";
+import { customElement, property } from "lit/decorators";
+import { fireEvent } from "../../../../../common/dom/fire_event";
+import "../../../../../components/ha-textfield";
+import { Action, SequenceAction } from "../../../../../data/script";
+import { haStyle } from "../../../../../resources/styles";
+import type { HomeAssistant, ItemPath } from "../../../../../types";
+import "../ha-automation-action";
+import type { ActionElement } from "../ha-automation-action-row";
+
+@customElement("ha-automation-action-sequence")
+export class HaSequenceAction extends LitElement implements ActionElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ type: Boolean }) public disabled = false;
+
+  @property({ attribute: false }) public path?: ItemPath;
+
+  @property({ attribute: false }) public action!: SequenceAction;
+
+  public static get defaultConfig() {
+    return {
+      sequence: [],
+    };
+  }
+
+  protected render() {
+    const { action } = this;
+
+    return html`
+      <ha-automation-action
+        .path=${[...(this.path ?? []), "sequence"]}
```

Comment:
This means the value is generated on each render, causing the element to think it has changed. It’s better to have a memoizeOne wrapped function that takes path and outputs path. It will return the exact same object each render.

Reply:
Agreed and added. Seems like this pattern can be added to a lot of more places.

../Frenck